setup.py

```diff
@@ -0,0 +1,15 @@
+from setuptools import setup, find_packages

```

Comment:
Not sure you need this python file. Can you remove it or explain why it's needed? It seems weird to have python files with the R package.